### PR TITLE
Add eks and kops for dependabot e2e labels

### PR DESCRIPTION
**Pull request by janisz**
## Description

Add labels to run on different configurations.
Refs: https://github.com/stackrox/stackrox/pull/1223

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### CODE REVIEW

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -28,10 +28,11 @@ updates:
...
     labels:
+      - "ci-aks-tests"
       - "ci-all-qa-tests"
+      - "ci-eks-tests"
       - "ci-no-api-tests"
       - "ci-no-ui-tests"
-      - "ci-eks-tests"
       - "ci-openshift-4-tests"
       - "ci-rhel-tests"
       - "dependencies"
```
Comment (gavin-stackrox) [inline .github/dependabot.yaml:38]: The upgrade test bounces between versions and runs a test not covered elsewhere.
```suggestion
      - "ci-upgrade-tests"
      - "dependencies"
```

**Review by gavin-stackrox — APPROVED** [review]

**Comment by janisz** [comment]
@gavin-stackrox should we remove kops from circleci config then?

**Comment by gavin-stackrox** [comment]
> @gavin-stackrox should we remove kops from circleci config then?

I don't see any kops in there

**Review by msugakov — APPROVED** [review]

**Comment by msugakov** [comment]
For the history. This relates to https://srox.slack.com/archives/C0321S70YK1/p1649759029278849

**Pull request merged**